src/lib/components/Avatar/Avatar.tsx

```diff
@@ -33,13 +32,13 @@ const AvatarComponent: FC<AvatarProps> = ({
   status,
   statusPosition = 'top-left',
   placeholderInitials = '',
+  className,
```

Comment:
Since it extends `ComponentProps<'div'>`, you don't need to specify the `className` property.